### ISSUE TITLE
#-1236012642------------- Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#892]
<strong>Source:</strong>`<div class="i-areastat-info-value">16</div>`
<strong>Element Location: </strong>.StyledListItem-sc-1sb8owe-1.feUFJp.sc-hAZoDl:nth-child(3) > .StyledListTextContainer-sc-1sb8owe-5.bgEsee > .StyledListSubtitle-sc-1sb8owe-4.jhpUkl.sc-dmRaPn > .sc-tsFYE.cFBREN.i-areastats-graphics > .sc-bhVIhj.bJVytO:nth-child(1) > .i-graphicstat > .i-flex.i-align-items-center > div > .i-graphicstat-info > .i-areastat-info-value
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix any of the following:
  Element has insufficient color contrast of 3.03 (foreground color: #949494, background color: #ffffff, font size: 10.0pt (13.328px), font weight: normal). Expected contrast ratio of 4.5:1
Tags: cat.color,wcag2aa,wcag143,ACT
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
